mainnet/jobs.json

```diff
@@ -78,5 +78,69 @@
     "name": "ethusd_cryptocompare",
     "selector": "[USD]",
     "url": "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_gemini",
+    "selector": "last",
+    "url": "https://api.gemini.com/v1/pubticker/btcusd"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_kucoin",
+    "selector": "data.BTC",
+    "url": "https://api.kucoin.com/api/v1/prices?base=USD&currencies=BTC"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_kraken",
+    "selector": "result.XXBTZUSD.c[0]",
+    "url": "https://api.kraken.com/0/public/Ticker?pair=BTCUSD"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_binance",
+    "selector": "price",
+    "url": "https://api.binance.com/api/v3/avgPrice?symbol=BTCBUSD"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_coinbase",
+    "selector": "data.amount",
+    "url": "https://api.coinbase.com/v2/prices/BTC-USD/spot"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_coingecko_api",
+    "selector": "bitcoin.usd",
+    "url": "https://api.coingecko.com/api/v3/simple/price?ids=bitcoin&vs_currencies=usd"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_bitfinex",
+    "selector": "[0]",
+    "url": "https://api-pub.bitfinex.com/v2/ticker/tBTCUSD"
```

Comment:
@super1114 noticed that `tBTC` is used here, wrapped/pegged tokens will not be accepted as valid sources to avoid any depegging or regulatory risks. This job can be removed.